### PR TITLE
French Translation :Update

### PR DESCRIPTION
**Pull request by MisterClems**
Missing line translation

### CODE REVIEW

File: lua/terrortown/lang/fr/marker.lua [hunk 1]
```diff
@@ -2,7 +2,7 @@ local L = LANG.GetLanguageTableReference("fr")
 
 -- GENERAL ROLE LANGUAGE STRINGS
 L[MARKER.name] = "Marqueur"
-L[TEAM_MARKER] = "TEAM Marqueur"
+L[TEAM_MARKER] = "TEAM du Marqueur"
```
Comment (TimGoll) [inline lua/terrortown/lang/fr/marker.lua]: take a look at the other languages, we swiched to "Team" instead of "TEAM"

**Review by TimGoll — CHANGES_REQUESTED** [review]

**Comment by TimGoll** [comment]
A general note, we switched from "The marker wins" to "Team Marker won. You might want to update this in French as well. But this is up to you

**Comment by MisterClems** [comment]
> A general note, we switched from "The marker wins" to "Team Marker won. You might want to update this in French as well. But this is up to you

Should i change  only for "hilite_win_ or for win_ too?

**Comment by TimGoll** [comment]
> > A general note, we switched from "The marker wins" to "Team Marker won. You might want to update this in French as well. But this is up to you
> 
> Should i change only for "hilite_win_ or for win_ too?

I think the latter isn't used anymore

**Comment by MisterClems** [comment]
> > > A general note, we switched from "The marker wins" to "Team Marker won. You might want to update this in French as well. But this is up to you
> > 
> > 
> > Should i change only for "hilite_win_ or for win_ too?
> 
> I think the latter isn't used anymore

okey i have just change for hilite_win all is good now

**Review by TimGoll — APPROVED** [review]

**Pull request merged**